### PR TITLE
Support Node v6 LTS and upgrade to Mapnik v3.x

### LINKED ISSUE #612: [node-v6] [mapnik-v3] Node v6 LTS and Mapnik v3.x support
Support for Node 0.10.x has ended in October 2016. We need to migrate to the latest Node LTS version (`v6.9.1`)
 
## TODO
 - [x] #611 Upgrade windshaft
 - [x] #610 Upgrade redis-mpool
 - [x] #609 Upgrade cartodb-psql 
 - [x] #608 Upgrade cartodb-redis
 - [x] #607 Upgrade camshaft

### PR DESCRIPTION
**Pull request by dgaubert**
Fixes #612

### CODE REVIEW

**Comment by dgaubert** [comment]
To read on Monday:

@rochoa, are we ready? 🎉  🎉  🎉

File: INSTALL.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ psql -d template_postgis -c 'CREATE EXTENSION postgis;'
 To fetch and build all node-based dependencies, run:
 
 ```
-npm install
+yarn [install]
```
Comment (rochoa) [inline INSTALL.md]: The `yarn install` command is deprecated, using `yarn` is the way to go.

**Comment by rochoa** [comment]
👍 .

File: INSTALL.md (hunk 1 above)
Comment (dgaubert) [inline INSTALL.md]: Woooops!

**Pull request merged**